modules/microshift-kubeconfig-overview.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_networking/microshift-kubeconfig.adoc
+
+:_content-type: CONCEPT
+[id="kubeconfig-files_{context}"]
+= Kubeconfig files
+
+The two categories of `kubeconfig` files used in {product-title} are local access and remote access. Every time {product-title} starts, a set of `kubeconfig` files for local and remote access to the API server are generated. These files are generated in the `/var/lib/microshift/resources/kubeadmin/` directory.
+
+The `kubeconfig` files are also examined for the latest Certificate Authorities (CAs). The CAs are tied to the certificates for the API server. The configuration of {product-title} certificates is determined by either default built-in values or a `config.yaml` file.
+//Q: anything else we should mention here about initial configuration values?
+
+[NOTE]
+====
+The`kubeconfig` command tree must exist for the cluster to be accessible. The values are applied from built-in default values or a 'config.yaml' if one was created.
```

Comment:
I'm not sure what "the kubeconfig command tree" is. Do you mean the set of kubeconfig files that MicroShift creates for the user?

Reply:
This might be a misunderstanding. Its the output of the `tree` command listing the contents of the kubeconfigs dir.